### ISSUE TITLE
Homepage / Roadmap Edits

### ISSUE THREAD

**Issue by briankelleher** [GreenhouseStudios/sourcery-web-app#436]
Roadmap:
```
<strong>Winter-Spring 2024</strong>
Release of ArchivesSpace plugin
Launch of Sourcery Anywhere (support for document requests nationwide)
Release of Android app
Release of "Request with Sourcery" website buttons
Support for Line Breaks and Basic Rich Text in Request Details

<strong>Summer-Fall 2024</strong>
Launch of minimal computing version for users in low-bandwidth areas.
Release of Tropy plugin
Support for institutional analytics

<strong>2025 and Beyond</strong>
Launch of in-app payments
Pilot launch of third-party, peer-to-peer scanning services
```

HOMEPAGE

How Sourcery Can Work for You
I am a RESEARCHER           I am an ARCHIVIST

How does Sourcery work?
The Sourcery app allows researchers to request scans of not-yet digitized documents from any archive through a single login and interface, accessible from any device. With Sourcery on a phone or laptop, a researcher seeking a document can simply enter the document's location and citation information and receive a scan directly within the app. It aims to provide better tools for communicating with archivists, easier payment options, tracking of requests placed at multiple institutions, and integration with popular research tools like Zotero and Tropy.
Who is behind Sourcery?
Sourcery emerges from the design process of Greenhouse Studios, a digital scholarship incubator a the University of Connecticut, and from the project team’s past work and ongoing interests in open source research software for the humanities. Sourcery is a project of the Corporation for Digital Scholarship, a not-for-profit organization.

[ARCHIVIST]
How does Sourcery work?
Sourcery—an open source, not-for-profit, web-based application for providing remote access to archival documents—aims to improve how researchers request document scans from archivists and provide archivists with better tools for fulfilling those requests. Sourcery provides archivists a streamlined, cross-platform (mobile, tablet, or laptop) reference scanning workflow, including tools to: help clarify, categorize, staff, and track outstanding requests; manage researcher communication; set and easily collect fees (for institutions that charge for these services); assess the true costs of their labor; and report these costs to decision makers in advocating for additional resources and staff.
Existing means of providing remote access to non-digitized sources create confusion for researchers and extra work for already stretched archivists. Sourcery aims to remedy these problems, creating new and more efficient pathways of computer-mediated research and systems to facilitate and properly account for and fund the often unaccounted and under-resourced labor of archivists in providing reference scans. At the same time, Sourcery recognizes that much archival research depends on close contact between researchers and archivists. By providing more effective online access, Sourcery ultimately aims to free up both researchers and archivists to focus on the sustained, mutual journeys of intellectual discovery at the heart of archival research.
What does Sourcery offer?
Manage requests for document scans and other researcher requests on any device from a simple, intuitive dashboard.
Fulfill requests directly from your phone or tablet or by snapping a photo in the stacks or upload a pdf, jpg, TIFF, or other file from your computer.
Share and allocate research requests among appropriate staff and keep track of who's doing what.
Ask and answer questions and make clarifications through the in-app, per-request Messaging feature. Declutter your email inbox and streamline correspondence right in the app.
See your institution's average turnaround time, most common file type requests, geographical areas of requests, new and repeat requesters, average file size, and more from the Institutional Analytics feature.
Who is behind Sourcery?
Sourcery emerges from the design process of Greenhouse Studios, a digital scholarship incubator a the University of Connecticut, and from the project team’s past work and ongoing interests in open source research software for the humanities. Sourcery is a project of the Corporation for Digital Scholarship, a not-for-profit organization.

Basically, I'd like users to come to the homepage and see two options after the top banner (I'm a researcher / I'm an archivist) and get different text based on their choice.

Then the login / register prompt would remain at the bottom

And the "Researcher" option could be the default display
